src/main/java/se/sundsvall/casestatus/service/mapper/SupportManagementMapper.java

```diff
@@ -12,16 +12,22 @@
 import se.sundsvall.casestatus.api.model.CaseStatusResponse;
 import se.sundsvall.casestatus.integration.db.SupportManagementStatusRepository;
 import se.sundsvall.casestatus.integration.db.model.SupportManagementStatusEntity;
+import se.sundsvall.casestatus.service.SupportManagementService;
 
 @Component
 public class SupportManagementMapper {
 
 	private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern(DATE_TIME_FORMAT);
 
 	private final SupportManagementStatusRepository supportManagementStatusRepository;
+	private final SupportManagementService supportManagementService;
+
+	public SupportManagementMapper(
```

Comment:
It feels a bit backwards to have a mapper that integrates with the backend layer and database. I don't know much about this service though and maybe there are reasons for having this pattern, but if possible then maybe look at the separation of conserns (i.e. let the mapper handle mapping and move the logic to call the backend to the layer above)?